resources/site.rb

```diff
@@ -38,10 +38,17 @@
     only_if "#{nginx_binary} -t"
   end
 
+  notify_group 'Delay nginx configuration validation' do
```

Comment:
I don't think notify_group was added until Chef 15 so this would require the `chef_version` to be bumped.

Reply:
fixed